airflow/ui/src/components/AppContainer/TimezoneDropdown.tsx

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import {
+  Box,
+  Button,
+  Menu,
+  MenuButton,
+  MenuList,
+  Tooltip,
+} from '@chakra-ui/react';
+import dayjs from 'dayjs';
+import tz from 'dayjs/plugin/timezone';
+import { getTimeZones } from '@vvo/tzdb';
+
+import Select from 'components/MultiSelect';
+import { useTimezoneContext } from 'providers/TimezoneProvider';
+
+dayjs.extend(tz);
+
+interface Option { value: string, label: string }
+
+const TimezoneDropdown: React.FC = () => {
+  const { timezone, setTimezone } = useTimezoneContext();
+  const [now, setNow] = useState(dayjs().tz(timezone));
+
+  const timezones = getTimeZones();
```

Comment:
Did you see this section of the tzdb docs https://github.com/vvo/tzdb/#notes:

```js
const value = timeZones.find((timeZone) => {
  return dbData.timeZone === timeZone.name || timeZone.group.includes(dbData.timeZone);
});
```

Reply:
Good catch. Should I include all group names in the dropdown too?

Reply:
I'll let you decide -- but I think that might be a bit noisy/large?

Reply:
That's what I thought too. I'll leave it as-is